README.md

```diff
@@ -24,6 +26,12 @@ make CMAKE_BUILD_TYPE=Release
 sudo make install
 ```
 
+Note, a neovim install from source on Mac OS may require <code>cmake</code> and <code>automake</code> to be installed first.
+
+```sh
+brew install cmake automake
+```
+
```

Comment:
the [build prereqs](https://github.com/neovim/neovim/wiki/Building-Neovim#macos--homebrew) (for mac at least) say to install more than what you list here -- do you know if these are enough?

Reply:
It seems like I had most of them installed (except for ninja). It's probably safer to include a link to the build prereqs 👍🏻